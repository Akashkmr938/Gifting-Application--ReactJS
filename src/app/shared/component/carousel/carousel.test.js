import React from 'react';
import renderer from 'react-test-renderer';
import Carousel from './carousel';

describe('The Carousel component', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<Carousel />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
})