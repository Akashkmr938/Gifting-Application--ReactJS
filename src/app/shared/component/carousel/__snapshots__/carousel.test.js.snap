// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Carousel component renders correctly 1`] = `
<div
  className="carousel-wrapper"
  style={
    Object {
      "height": "40%",
      "marginBottom": "60px",
      "width": "100%",
    }
  }
>
  <span
    id="target-item-1"
  />
  <span
    id="target-item-2"
  />
  <span
    id="target-item-3"
  />
  <div
    className="carousel-item item-1"
    style={
      Object {
        "backgroundImage": "url('https://image.freepik.com/free-vector/black-friday-gift-sale-minimal-background_1017-15996.jpg')",
        "height": "450px",
        "marginBottom": "20px",
      }
    }
  />
</div>
`;
